src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -1522,13 +1522,10 @@ private static FileTypeDetector createDefaultFileTypeDetector() {
 
         // loads all installed file type detectors
         private static List<FileTypeDetector> loadInstalledDetectors() {
-            List<FileTypeDetector> list = new ArrayList<>();
-            ServiceLoader<FileTypeDetector> loader = ServiceLoader
-                .load(FileTypeDetector.class, ClassLoader.getSystemClassLoader());
-            for (FileTypeDetector detector: loader) {
-                list.add(detector);
-            }
-            return list;
+            return ServiceLoader.load(FileTypeDetector.class,
+                                      ClassLoader.getSystemClassLoader())
+                .stream()
+                .map(p -> p.get()).toList();
```

Comment:
Let's move `.toList();` to a separate line

Reply:
So changed in [44401e7](https://github.com/openjdk/jdk/pull/22552/commits/44401e788d6cd9f3b67a0ba8a67874e8aca7ba6e).